README.md

```diff
@@ -51,7 +69,7 @@ Find a bug or want to request a new feature?  Please let us know by submitting a
 Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
 
 ## Licensing
-Copyright 2017 Esri
+Copyright 2017 - 2019 Esri
```

Comment:
Do we update the Copyright range or just stick with 2017?

Reply:
@dg0yal said it was OK to use the range.

Reply:
Perfect.